src/main/java/ai/rev/speechtotext/models/asynchronous/RevAiJobOptions.java

```diff
@@ -44,6 +44,10 @@
   @SerializedName("metadata")
   private String metadata;
 
+  /** Optional parameter to filter profanity in the transcript */
+  @SerializedName("filter_profanity")
```

Comment:
I think it makes sense to add these in a separate PR.
Landing logically separate improvements as separate commits allows us to easily rollback changes with git cli and provides a better git history with clear commit messages.

Reply:
for example: we could also add remove_disfluencies option, but I think it would be best if that happened in a separate commit

Reply:
But if filter_profanity is already implemented in this pr then no reason to delay by taking it out. we'll just keep it in mind for the future.